test/test_core.py

```diff
@@ -8600,6 +8638,7 @@ def test_wasm2js(self):
   @no_asan('no wasm2js support yet in asan')
   @requires_wasm2js
   @also_with_minimal_runtime
+  @no_big_endian('wasm2js is currently not compatible with big endian')
```

Comment:
Can you add a line after this line https://github.com/emscripten-core/emscripten/blob/7ed425edb82161c3d60ef5d00f939446edddce42/tools/link.py#L847 inside `if settings.SUPPORT_BIG_ENDIAN:`

```py
if settings.WASM2JS:
  exit_with_error('WASMJ2S is currently not compatible with SUPPORT_BIG_ENDIAN')
```